nlmod/dims/layers.py

```diff
@@ -678,7 +730,7 @@ def combine_layers_ds(
     ds_combine = xr.Dataset(da_dict, attrs=attrs)
 
     # remove layer dimension from top again
-    ds = remove_layer_dim_from_top(ds, inconsistency_threshold=0.001)
+    ds = remove_layer_dim_from_top(ds, inconsistency_threshold=1e-3)
```

Comment:
this is not the one that is returned, so I guess the layer dimension is not removed

Reply:
In place maybe?

Reply:
I think the code was modified, but the message never was, so I've changed it now. The layer dim is actually removed from the top in the returned dataset.